src/settings/SettingsDetails.js

```diff
@@ -21,7 +21,7 @@ export default class SettingsDetails extends Component {
 					<LabeledText
 						id="name"
 						label="Name"
-						value={settings.name}
+						value={settings.name || ''}
```

Comment:
@borisyankov this checking is required somewhere since it's null, I think placing them in SettingsDetails is better, as it's within context so we can have a little inconsistency.

What do you think?
it's return from backend as null, so I think we better keep it this way in store.